src/integration-test/java/uk/gov/ida/verifylocalmatchingserviceexample/MatchingServiceResourceTest.java

```diff
@@ -89,6 +90,70 @@ public void shouldReturnMatchWhenUserWithCaseInsensitiveSurnameDateOfBirthPostco
         assertEquals(MATCH, response.readEntity(MatchStatusResponseDto.class));
     }
 
+    @Test
+    public void shouldReturnMatchWhenOneUserMatchFoundForCycle3Attribute() throws JsonProcessingException {
+        LocalDate dateOfBirth = LocalDate.of(1984, 6, 16);
+        String firstName = "some-first-name";
+        String surname = "some-surname";
+        String postcode = "some-postcode";
+
+        testDatabaseRule.ensureUserWithAddressExist(surname, firstName, dateOfBirth, postcode);
+        testDatabaseRule.ensureUserWithAddressExist(surname, firstName, dateOfBirth, postcode);
```

Comment:
Duplicated line?

Reply:
Thats because we want two users with same info except for national insurance number